/*
 * Copyright Â© 2023- Gimlet Labs, Inc.
 * All Rights Reserved.
 *
 * NOTICE:  All information contained herein is, and remains
 * the property of Gimlet Labs, Inc. and its suppliers,
 * if any.  The intellectual and technical concepts contained
 * herein are proprietary to Gimlet Labs, Inc. and its suppliers and
 * may be covered by U.S. and Foreign Patents, patents in process,
 * and are protected by trade secret or copyright law. Dissemination
 * of this information or reproduction of this material is strictly
 * forbidden unless prior written permission is obtained from
 * Gimlet Labs, Inc.
 *
 * SPDX-License-Identifier: Proprietary
 */

syntax = "proto3";

package gml.gem.calculators.tensorrt.optionspb;

option go_package = "gimletlabs.ai/gimlet/src/gem/calculators/plugin/tensorrt/optionspb;optionspb";

message TensorRTExecuteCalculatorOptions {
  // Ordered onnx names. Ordering of names is used for matching onnx inputs/outputs to the correct
  // mediapipe input/output streams.
  repeated string input_onnx_name = 1;
  repeated string output_onnx_name = 2;
}
